sklearn/utils/fixes.py

```diff
@@ -343,6 +343,11 @@ def __call__(self, *args, **keywords):
     from functools import partial
 
 
+def parallel_helper(obj, methodname, *args, **kwargs):
```

Comment:
It is used in `forest.py`and `multioutput.py`, so the thinking was to move it to a common place instead of duplicating the code.